static/css/main.css

```diff
@@ -122,3 +122,31 @@ table tr th, table tr td {
         0 0 0 2.1em #212529,
         0 0 0 2.6em #95a5a6;
 }
+
+.login {
+    background-color: var(--light, #eee);
+}
+.login main {
+    max-width: 330px;
+    margin: 40px auto;
+}
+
+meter.progress {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    background-image: none;
+    width: 100%;
+}
+meter.progress::-webkit-meter-inner-element {
+    display: block;
+}
+meter.progress::-webkit-meter-bar {
+    border: 0;
+    border-radius: 0;
+}
+meter.progress::-webkit-meter-optimum-value {
+    background: var(--primary);
+}
+meter.progress::-moz-meter-bar {
+    background: var(--primary);
+}
```

Comment:
This is all hacky and brittle. IMHO we could also use `<meter>` without bootstrap styling.

Reply:
How hacky and brittle is it? Like in what way does it break? I think if it mostly works and "breaks" by falling back to a more default styling, that would probably be okay?

Reply:
I think it is unlikely that this breaks in a way that affects other elements on the page, but it could well be that the relevant part of the bar is not colors.

I would feel safer with reverting d2060da325e0a20583261a38fb4b551cd88b671d. However, other people tent to value esthetics higher than I do.

Reply:
I'll try to also test it in safari and maybe edge. If it also works there, I'd just go ahead with it and deploy it. the bar isn't used a lot and also has numbers on it, so even if it breaks it would be noticeable and we could still make changes.